src/System.Data.SqlClient/src/Interop/LocaleMapperApiHelper.cs

```diff
@@ -0,0 +1,482 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Can we leave the LocaleInterop.cs as is and use a naming convention like the files at the link below for the new non-windows files you are adding? 
https://github.com/dotnet/corefx/tree/master/src/System.Runtime.InteropServices.RuntimeInformation/src